nss-c31-mixed-havoc/nutshell
# Js articles

# Description
The articles module

Applies to issue # 4

## Type of change

Please delete options that are not relevant.



# Testing Instructions

git fetch --all
git checkout js-articles
cd src/lib run grunt
go to localhost:8080
check to make sure articles show after login 
make sure you can delete,edit and create new articles
make sure articles of your friends show up





# Checklist:
